website/blog/2023-04-14-release-0.14.md

```diff
@@ -0,0 +1,110 @@
+The installed `kind` CLI is available from the system shell [1516](https://github.com/containers/podman-desktop/issues/1516).
+
+### Kind Clusters in Resources
+
+[Create Kind clusters from an updated **Settings > Resources** page](/docs/kubernetes/kind/creating-a-kind-cluster) that also allows you to start/stop [1953](https://github.com/containers/podman-desktop/issues/1953) and delete [1977](https://github.com/containers/podman-desktop/issues/1977) them.
+
+### Share Images with Kind
+
+It is easy to move images from Podman to Kind [1448](https://github.com/containers/podman-desktop/issues/1448).
+
+### Ingress
+
+If you deploy a pod to Kind, you are going to want to reach it! We have made it easy to install an ingress
+controller (Contour) [1675](https://github.com/containers/podman-desktop/issues/1675).
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/695993/230328543-71a6d2a4-b6ac-44dd-9219-d0a864dd44f9.gif" />
+
+### Play YAML on Kubernetes
+
+Easily deploy local YAML files to your Kind or other Kubernetes clusters [1261](https://github.com/containers/podman-desktop/issues/1261).
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/231812563-ece0a56a-b347-48f8-a3a7-400eb9449037.mp4" />
+
+### UI and UX improvements
+
+#### Updated Preferences
+
+The **Settings > Preferences** page has been updated with a new design [1913](https://github.com/containers/podman-desktop/pull/1913),
+making it easier to see and change preferences. Changes are live, no more Update button.
+
+![preferences](https://user-images.githubusercontent.com/49404737/229498507-e754b55c-dcbd-486d-9ee3-a1fe3bed7271.gif)
+
+#### Telemetry Prompt
+
+The prompt to enable or disable telemetry has been moved from its own dialog into the **Welcome** screen.
+[1927](https://github.com/containers/podman-desktop/pull/1927)
+This is more usable, one less prompt, and solves a window-layering issue for some users!
+
+![telemetry prompt](https://user-images.githubusercontent.com/19958075/229577331-365a9a01-0426-4482-a95d-f5dfe39af90a.png)
+_________________
+
+## Other notable enhancements
+
+- Kubernetes pods are shown in the **Pods** screen [1263](https://github.com/containers/podman-desktop/issues/1263)
+- Extension support for opening an external URL [2028](https://github.com/containers/podman-desktop/pull/2028) and
+  accessing the clipboard [2023](https://github.com/containers/podman-desktop/pull/2023)
+_________________
+
+## Documentation
+
+Naturally, we have a section in the documentation just for [Kind](https://podman-desktop.io/docs/kubernetes/kind).
+
+_________________
+
+## Notable bug fixes
+
+- Avoid a dialog box if unable to check for updates [2062](https://github.com/containers/podman-desktop/pull/2062)
+- Unable to get to the Dashboard if Kind (or Kubernetes) cluster was not running [2052](https://github.com/containers/podman-desktop/issues/2052)
+
+
```

Comment:
No community thank you?

Reply:
This release was only a bit over a week long and included the Easter weekend, so we had no external contributions.